ci_framework/roles/os_net_setup/tasks/main.yml

```diff
@@ -1,3 +1,31 @@
+- name: pre-debug oc get clouds_yaml or keystone commands
+  failed_when: false
+  register: debug_oc_get
+  shell:
+    cmd: |
+      set -x
+
+      eval $(${HOME}/ci-framework-data/bin/crc oc-env)
+      export KUBECONFIG="${HOME}/.crc/machines/crc/kubeconfig"
+
+      ons_clouds_yaml=$(oc get configmap -n openstack openstack-config -o 'jsonpath={.data.clouds\.yaml}')
+      ons_secret=$(oc get secret -n openstack openstack-config-secret -o 'jsonpath={.data.secure\.yaml}')
+
+      KEYSCT=$(oc get keystoneapi keystone -o json | jq -r .spec.secret)
```

Comment:
In the gate, I noticed `jq` was unavailable. You may want to think about  using it. This could help
```
oc get keystoneapi keystone -o jsonpath='{ .spec.secret}'
```

Reply:
Likewise in the below lines... we could opt to use `-o jsonpath` instead. This would avoid the dependency on `jq`.

Reply:
thanks good point, yes using jsonpath seems way better, no need for extra command/processing afterwards if we can query the exact detail directly